pages/api/studentPortal/quizzes/scoringLogicFPL.ts

```diff
@@ -0,0 +1,26 @@
+import { LanguageQuizResultsMap } from "../../../../components/resources/quizzes/langQuiz/LanguageResults";
+import { QuizViewState } from "../../../../components/resources/quizzes/shared/types";
+
```

Comment:
looks like you have multiple files in the quizzes folder that are related to your quiz, I would create a subfolder for your quiz and move your files into that folder